global/scripts/debug.shlib

```diff
@@ -0,0 +1,261 @@
+# Description:
```

Comment:
For shlibs, I have started using a convention of `#!/bin/echo This script should not run directly:` for the first line, so that if someone were to make it executable and run it from a modern shell, it does this:

```
tim@timsdev:/mnt/myelin/tim/pipelinedevel/Pipelines/global/scripts$ chmod +x newopts.shlib 
tim@timsdev:/mnt/myelin/tim/pipelinedevel/Pipelines/global/scripts$ ./newopts.shlib
This script should not be run directly: ./newopts.shlib
```

I think shlibs should not have execute permissions, but I have found some in the pipelines that did, so as an extra layer of safety...

Reply:
Added.